app/controllers/content_items_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ContentItemsController < ApplicationController
   def index
     @organisation = Organisation.find(params[:organisation_id])
-    @content_items = @organisation.content_items.first(25)
+    @content_items = @organisation.content_items.order("#{params[:sort]} #{params[:order]}").first(25)
```

Comment:
It might be a good idea to use [strong parameters](https://github.com/rails/strong_parameters)